GNUmakefile

```diff
@@ -66,5 +66,8 @@ website-lint-fix:
 	@misspell -w -source=text website/
 	@terrafmt fmt ./website --pattern '*.markdown'
 
+website-list-category:
```

Comment:
Is that related to the deprecation? Or just piggybacking?

Reply:
Sort of related - this verifies website categories are consistent. I'll add it to commit description